src/deps_log.cc

```diff
@@ -206,6 +204,7 @@ LoadStatus DepsLog::Load(const string& path, State* state, string* err) {
       read_failed = true;
       break;
     }
+    offset += size + 4;
```

Comment:
why 4?

Reply:
fread() reads first 4 bytes (line 196), then later `size` bytes.

Reply:
Ah right. Can you change this to `offset += sizeof(size) + size;` and also replace the 4 above to make this clearer? Maybe even change `unsigned size;` to `uint32_t size;`.

Reply:
Done.